tests/e2e/config_secret.go

```diff
@@ -136,9 +141,9 @@ var _ = ginkgo.Describe("Config-Secret", func() {
 		ginkgo.By("Create testuser1 and assign required roles and privileges to testuser1")
 		createTestUserAndAssignRolesPrivileges(masterIp, configSecretTestUser1,
 			configSecretTestUser1Password, configSecretUser1Alias, propagateVal,
-			dataCenters, clusters, hosts, vms, datastores)
+			dataCenters, clusters, hosts, vms, datastores, "createUser", "createRoles")
```

Comment:
I prefer to recreate user than this which will help in case a failed test leaves stale users

Reply:
I am performing cleanup of testusers, roles/privileges assigned to testusers before the start of any testcase.